docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3.7'
+
+services:
+  web:
+    build: ./
+    command: python manage.py runserver 0.0.0.0:8000
+    volumes:
+      - ./:/usr/src/app/
+    ports:
+      - 8000:8000
+    env_file:
+      - .envs/.local/.env.dev
+
+  db:
+    image: postgres:12.0-alpine
+    volumes:
+      - postgres_data:/var/lib/postgresql/data/
+    environment:
+      - POSTGRES_USER=ikarikun
+      - POSTGRES_PASSWORD=getonthefuckingrobot
+      - POSTGRES_DB=new_tokyo_03
```

Comment:
You should use the env variables here too, to make sure the values from `db` and `web` always match